README.md

```diff
@@ -62,49 +67,71 @@ A Complete (nearly) mod to have various features on various Minecraft versions
 
-```
+## Building from Source
+
+For those interested in the most current version or building from the source directly, follow the instructions below:
+
+### Basic Build
+
+To build the project, use the following command:
+
+```bash
 ./gradlew build
 ```
 
-Use `-Paxolotlclient.modules.<version_name>=true` to add a version to the build. Add `-Paxolotlclient.modules.all=true`
-to build everything. In case you use an IDE, you can also add the respective properties to `gradle.properties` files
-(or remove them).
-
-### Licensing
+### Building Specific Versions
 
-- This mod is licensed under the LGPL-3.0 License.
+If you wish to build a specific version, use:
 
+```bash
+./gradlew build -Paxolotlclient.modules.<version_name>=true
 ```
-AxolotlClient-mod
-Copyright (C) 2021-present moehreag + Contributors
-
-This program is free software; you can redistribute it and/or
-modify it under the terms of the GNU Lesser General Public
-License as published by the Free Software Foundation; either
-version 3 of the License, or (at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-Lesser General Public License for more details.
-
-You should have received a copy of the GNU Lesser General Public License
-along with this program; if not, write to the Free Software Foundation,
-Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+Replace `<version_name>` with the desired version name.
+
+### Building All Versions
+
+To build all available versions at once, use:
+
+```bash
+./gradlew build -Paxolotlclient.modules.all=true
 ```
 
-- For more information see the [LICENSE](LICENSE) file.
+### IDE Tips
+
+For those using an Integrated Development Environment (IDE), you can add the respective properties mentioned above to the `gradle.properties` files or remove them as required.
+
+## API Requirements
+
+Ensure to download the respective API package for your version before you download:
+
+- 1.8.9: Legacy Fabric API
+- 1.16.x: Fabric API (necessary parts are shipped with the mod)
+- 1.19+: QSL (necessary parts are shipped with the mod)
+
+## Disclaimers
+
+We do not endorse or support cheats/hacks. If you believe any options packaged by this mod could be considered as such, please let us know.
+
+The only data sent to our server is your public Minecraft account UUID, which isn't stored longer than one game session.
+
+## Credits
+
+Our heartfelt thanks to the open-source contributors, particularly DarkKronicle, the creator of KronHUD, and AMereBagatelle, the author of the FabricSkyBoxes mod. Without their contributions, Axolotl Client would not be as comprehensive as it is.
```

Comment:
This is slightly misleading since at that point neither DarkKronicle nor AMereBagatelle contributed to the mod, and only DarkKronicle did now (for something entirely different).

Reply:
Maybe also add hyperlinks to the respective mods?

Reply:
yes, if you don't have them grab them off the  modrinth page